deploy/helm/sumologic/README.md

```diff
@@ -389,7 +391,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `otelevents.persistence.pvcLabels`                                        | Additional PersistentVolumeClaim labels for all OpenTelemetry Collector pods.                                                                                                                                                                                                                                                                                        | `{}`                                                                                                              |
 | `otelevents.config.override`                                              | Override configuration for OpenTelemetry Collector. See [the documentation](../../docs/opentelemetry_collector.md#customizing-opentelemetry-collector-configuration) for more details.                                                                                                                                                                               | `{}`                                                                                                              |
-| `otelevents.statefulset`                                                  | OpenTelemetry Collector StatefulSet customization options. See values.yaml for more details.                                                                                                                                                                                                                                                                         | See values.yaml                                                                                                   |                                                                                                                                                                                                                                                                                              | `Nil`                                                                                                             |                                                                                                                                                                                                                                                                                                           | `Nil`
+| `otelevents.statefulset`                                                  | OpenTelemetry Collector StatefulSet customization options. See values.yaml for more details.                                                                                                                                                                                                                                                                         | See [values.yaml]                                                                                                   |
```

Comment:
There was two additional spaces after my suggestion

```suggestion
| `otelevents.statefulset`                                                  | OpenTelemetry Collector StatefulSet customization options. See values.yaml for more details.                                                                                                                                                                                                                                                                         | See [values.yaml]                                                                                                 |
```

Reply:
Oh, I missed this suggestion but I think I made a similar change just now and I still get formatting error, I ran this through my markdown formatter.